client/build.gradle

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+apply plugin: 'java'
+apply plugin: 'maven-publish'
+apply plugin: 'jacoco'
+apply plugin: 'io.freefair.lombok'
+
+allprojects {
+    group = opensearch_group
+    version = "${opensearch_build}"
+    targetCompatibility = JavaVersion.VERSION_11
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
is this compatibility correct? I thought whole opensearch moved to java 17.

Reply:
Actually I think that make more sense to enforce the same from `opensearch.build` instead, in this way the Java version requirement of this module will always identical to OpenSearch main repo.

Reply:
I replaced above with `apply plugin: 'opensearch.build'` instead.